.ncurc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "reject": [
+    "async",
+    "sinon"
```

Comment:
@elnyry FYI. Need a story update these dependencies. There are a lot of failed tests when doing so.